app.py

```diff
@@ -394,24 +396,45 @@ def load_img(img_name):
+    <html>
+        <head>
+            
+        </head>
+
+        <body>
+            <a>
+                <strong style="font-size:20px">
+                    <!-- 1khanfarhan10 &nbsp;&nbsp;-->
+                    <pre class="tab">1khanfarhan10 <a style="font-size:14px">26 commits </a><a style="color: #2bff00;font-size:10px">18,069++ </a><a style="color: #FF0000;font-size:10px">9,501--</a>   soumya997 <a style="font-size:14px">3 commits </a><a style="color: #2bff00;font-size:10px">19,938++ </a><a style="color: #FF0000;font-size:10px">3,460--</a></pre>
+                    <div class="github-card" data-github="khanfarhan10" data-width="350" data-height="150" data-theme="default"></div>
+                    <script src="//cdn.jsdelivr.net/github-cards/latest/widget.js"></script>
+                    <div class="github-card" data-github="soumya997" data-width="350" data-height="" data-theme="default"></div>
+                    <script src="//cdn.jsdelivr.net/github-cards/latest/widget.js"></script>
+                </strong> 
+                <br>
+                <strong style="font-size:20px">
+                    <!-- 1khanfarhan10 &nbsp;&nbsp;-->
+                    <pre class="tab">Nibba2018 <a style="font-size:14px">2 commits </a><a style="color: #2bff00;font-size:10px">25++ </a><a style="color: #FF0000;font-size:10px">27--</a>           Dsantra92 <a style="font-size:14px">1 commits  </a><a style="color: #2bff00;font-size:10px">1,129++ </a><a style="color: #FF0000;font-size:10px">0--</a></pre>
+                    <div class="github-card" data-github="Nibba2018" data-width="350" data-height="" data-theme="default"></div>
+                    <script src="//cdn.jsdelivr.net/github-cards/latest/widget.js"></script>
+                    <div class="github-card" data-github="Dsantra92" data-width="350" data-height="" data-theme="default"></div>
+                    <script src="//cdn.jsdelivr.net/github-cards/latest/widget.js"></script>
+                </strong>
+                <br> 
+                <strong style="font-size:20px">
+                    <!-- 1khanfarhan10 &nbsp;&nbsp;-->
+                    <pre class="tab">debangeedas <a style="font-size:14px">1 commits </a><a style="color: #2bff00;font-size:10px">40++ </a><a style="color: #FF0000;font-size:10px">0--</a>          BALaka-18 <a style="font-size:14px">1 commits </a><a style="color: #2bff00;font-size:10px">5++ </a><a style="color: #FF0000;font-size:10px">4--</a></pre>
+                    <div class="github-card" data-github="debangeedas" data-width="350" data-height="" data-theme="default"></div>
+                    <script src="//cdn.jsdelivr.net/github-cards/latest/widget.js"></script>
+                    <div class="github-card" data-github="BALaka-18" data-width="350" data-height="" data-theme="default"></div>
+            <script src="//cdn.jsdelivr.net/github-cards/latest/widget.js"></script>
+                </strong> 
```

Comment:
Looks like you are hard coding the information here. We want it to be dynamic so that new changes are visible as well.

Reply:
yes, it's hard coded, please show me some tutorial as I have very less experience of doing things dynamically,
The things you are seeing, it took me 2 to 3 continuous hours to figure that out.
Or you may go for a slightly experienced person to handle that.

Reply:
> please show me some tutorial as I have very less experience of doing things dynamically
I asked you to refer this(https://github.com/fury-gl/fury/blob/master/docs/source/ext/github_tools.py) for this reason.

@soumya997 Okay this can be done on a separate PR.

Reply:
I will try it soon, but for now, it will be good for me to skip that part. Thank you for suggesting that @Nibba2018 .

Reply:
I tried some StackOverflow code too but did not work for me. This was the code, take a look if time permits
```from requests import session
def login():
    sess = session()

    # login here and return session with valid creds
    return sess

def generateList(link):
    # you need to login before you do anything
    sess = login()

    # because of the way that requests works, you must start out by creating an object to
    # imitate the response object. This will help you to cleanly while-loop through
    # github's pagination
    class response_immitator:
        links = {'next': {'url':link}}
    response = response_immitator() 
    while 'next' in response.links:
        response = sess.get(response.links['next']['url'])
        for repo in response.json():
            yield repo

def check_commit_count(baseurl, user_name, repo_name, max_commit_count=None):
    # login first
    sess = login()
    if max_commit_count != None:
        totalcommits = 0

        # construct url to paginate
        url = baseurl+"repos/" + user_name + '/' + repo_name + "/stats/contributors"
        for stats in generateList(url):
            totalcommits+=stats['total']

        if totalcommits >= max_commit_count:
            return False
        else:
            return True

def main():
    # what user do you want to check for commits
    user_name = "soumya997"

    # what repo do you want to check for commits
    repo_name = "AI-Hacktoberfest"

    # github's base api url
    baseurl = "https://api.github.com/"

    # call function
    check_commit_count(baseurl, user_name, repo_name,30)

if __name__ == "__main__":
    main()
```

Reply:
@soumya997 , No problem we will create a separate issue for this. There's already a lot of work being done in this PR.